docs/book/src/bytecode/index.md

```diff
@@ -38,12 +31,41 @@ Once imported, using the Bytecode Library is as simple as calling the desired fu
 - `verify_predicate_address()`
 - `verify_predicate_address_with_configurables()`
 
+## Known Issues
+
+Please note that if you are passing the bytecode from the SDK and are including configurable values, the `Vec<u8>` bytecode provided must be copied to be mutable. The following can be added to make your bytecode mutable:
+
+```sway
+fn foo(not_mutable_bytecode: Vec<u8>) {
```

Comment:
This code doesn't compile for me on the latest toolchain. Can you import it from an example somewhere so there is a check on it? 

Also, I'm guessing the comment at the end should say "You may now use" instead of "You may not used"?

Reply:
This compiles on the version of forc that this repo is using. I will update this in https://github.com/FuelLabs/sway-libs/pull/226 and move things into separate files in another PR

Reply:
https://github.com/FuelLabs/sway-libs/issues/228